app/config/collections.php

```diff
@@ -1158,7 +1158,18 @@
                 'filters' => [],
             ],
             [
-                '$id' => 'providerToken',
+                '$id' => 'providerAccessToken',
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 16384,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => ['encrypt'],
+            ],
+            [
+                '$id' => 'providerRefreshToken',
```

Comment:
Let's add `providerAccessTokenExpiry` with a unix timestamp as the value.

Reply:
Done.